x-pack/solutions/observability/plugins/synthetics/docs/openapi/synthetic_apis.yaml

```diff
@@ -0,0 +1,1446 @@
+                    "tags": ["authentication", "security"],
+                    "share_across_spaces": true
+                  }
+              postParametersRequestExample2:
+                summary: Multiple parameters
+                description: Add multiple parameters.
+                value: |-           
+                  [
+                    {
+                      "key": "param1",
+                      "value": "value1"
+                    },
+                    {
+                      "key": "param2",
+                      "value": "value2"
+                    }
+                  ]
+      responses:
+        '200':
+          description: A successful response.
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - type: array
+                    items:
+                      $ref: "#/components/schemas/postParameterResponse"
+                  - $ref: "#/components/schemas/postParameterResponse"
+              examples:
+                postParametersResponseExample1:
+                  summary: Single parameter
+                  description: A successful response for a single added parameter.
+                  value: |-
+                    {
+                      "id": "unique-parameter-id",
+                      "key": "your-key-name",
+                      "value": "your-param-value",
+                      "description": "Param to use in browser monitor",
+                      "tags": ["authentication", "security"],
+                      "share_across_spaces": true
+                    }
+                postParametersResponseExample2:
+                  summary: Multiple parameters
+                  description: A successful response for multiple added parameters.
+                  value: |-
+                    [
+                      {
+                        "id": "param1-id",
+                        "key": "param1",
+                        "value": "value1"
+                      },
+                      {
+                        "id": "param2-id",
+                        "key": "param2",
+                        "value": "value2"
+                      }
+                    ]
+  /api/synthetics/params/{id}:
+    delete:
+      summary: Delete a parameter
+      operationId: delete-parameter
+      description: >
+        Delete a parameter from the Synthetics app.
+        You must have `all` privileges for the Synthetics feature in the Observability section of the Kibana feature privileges.
+      tags:
+        - synthetics
+      parameters:
+        - in: path
+          name: id
+          description: The ID for the parameter to delete.
+          required: true
+          schema:
+            type: string
+    get:
+      summary: Get a parameter
+      operationId: get-parameter
+      description: You must have `read` privileges for the Synthetics feature in the Observability section of the Kibana feature privileges.
```

Comment:
```suggestion
      description: Get a parameter from the Synthetics app.
      
      You must have `read` privileges for the Synthetics feature in the Observability section of the Kibana feature privileges.
```

Reply:
Thanks! Fixed in fa994557ff351594bf7cbdccf473bea646391109 since I needed to add `>` or `|` first.